responses/registries.py

```diff
@@ -73,6 +79,31 @@ def replace(self, response: "BaseResponse") -> "BaseResponse":
         self.registered[index] = response
         return response
 
+    def _dump(self, destination: "io.IOBase") -> None:
```

Comment:
I decided to create it in the registry because this will allow to dump existing tests to the file
Eg
If user ran `responses.add()` multiple times, then user can run `responses.get_registry()._dump(*args)`

That will allow safe (compared to brand new recording) transfer to the new method if required

In the future we can provide `serializer` argument, but at the moment I am a bit reluctant to support other serializers than one we decide

Also, user can also overwrite this method after inheriting

Or what are the other options to introduce this method?
As a separate function?

Reply:
Couldn't dumping be on the `Recorder()`?  I'm concerned that recording feels like it is half in the Recorder and half in the registry right now.

Reply:
@markstory 
done!

this could be done even as a separate function. Then it will be possible to easily dump existing registries into the file